app/main.py

```diff
@@ -2,0 +1,34 @@
+import os
+
+
+def move_file(string: str) -> None:
+    words = string.split()
+    if len(words) != 3:
+        raise TypeError
+    elif words[0] != "mv":
+        raise ValueError
+
+    old_file_name = words[1]
+
+    if words[-1][-1] == "\\" or words[-1][-1] == "/":
+        destination_path = os.path.join(*words[-1])
+        os.makedirs(destination_path, exist_ok=True)
+        return None
```

Comment:
Returning `None` here will stop the function execution, which means the file won't be moved. Consider revising this logic to ensure the file is moved to the directory.